blockchain/blockchain.go

```diff
@@ -1145,50 +1145,54 @@ func (b *Blockchain) Finalise(pending *Pending, sign BlockSignFunc) error {
 	})
 }
 
-func (b *Blockchain) GenesisState() (*core.StateUpdate, error) {
-	var genesisStateUpdate *core.StateUpdate
-	return genesisStateUpdate, b.database.View(func(txn db.Transaction) error {
-		var err error
-		genesisStateUpdate, err = genesisState(txn)
-		return err
-	})
-}
-
-func genesisState(txn db.Transaction) (*core.StateUpdate, error) {
-	var genesisStateUpdate *core.StateUpdate
-	return genesisStateUpdate, txn.Get(db.GenesisState.Key(), func(b []byte) error {
-		genesisStateUpdate = new(core.StateUpdate)
-		return encoder.Unmarshal(b, genesisStateUpdate)
-	})
-}
+func (b *Blockchain) StoreGenesis(diff *core.StateDiff, classes map[felt.Felt]core.Class) error {
+	receipts := make([]*core.TransactionReceipt, 0)
+	block := &core.Block{
```

Comment:
The genesis block has pre-defined fields for now, so it only needs to be defined in one place. If we want different genesis blocks for different networks, we can expand the genesis config and let the `genesis` package build the block.

Because the genesis config doesn't have any influence over the genesis block, I think leaving the block in `StoreGenesis` makes sense.